docs/team/noelmathewisaac.md

```diff
@@ -0,0 +1,63 @@
+---
+
+## Project: Taskify
+
+Taskify is a desktop task management application for university students. Using Taskify, students can
+manage all their tasks (academics/personal/CCAs) effectively and seamlessly. The user interacts with it using a CLI,
+and it has a GUI created with JavaFX. It is written in Java, and has about 12 kLOC.
+
+Given below are my contributions to the project.
+
+* ### Major Contributions:
+    * #### Implemented the tag-search feature
+        * What it does: Allows the users to search for a task based on one or more tags.
+        * Justification: This feature makes it much easier for the user to categorise and filter out tasks belonging to a
+          certain category. By tagging tasks with relevant keywords, the user can later retrieve all the tasks from 
+          the same
+          cateogory really quickly instead of manually looking through all the tasks.
+        * Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
+        * Relevant PRs: [\#36](https://github.com/AY2021S2-CS2103T-W14-4/tp/pull/36)
+    * #### UI Design and Enhancement
+      * What it does: Designed and enhanced the UI to make it more seamless and user friendly, while also being 
+        simple and functional.
+      * Justification: A great user interface is essential to the user expereince and will help in user retention 
+        and overall motivation to use the app.
+      * Highlights: The UI design involved getting feedback from multiple user. The color scheme and layout of the 
+        UI was designed to be simple and intuitive. The implementation of the tab feature was quite challenging and 
+        involved a lot of research, especially on aspects like shape and dynamic resizing.
+      * Relevant PRs: [\#56](https://github.com/AY2021S2-CS2103T-W14-4/tp/pull/56), 
+        [\#86](https://github.com/AY2021S2-CS2103T-W14-4/tp/pull/86), [\#89](https://github.com/AY2021S2-CS2103T-W14-4/tp/pull/89), [\#93](https://github.com/AY2021S2-CS2103T-W14-4/tp/pull/93)
+
+
+
+* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2021s2.github.io/tp-dashboard/?search=noelmathewisaac&sort=groupTitle&sortWithin=title&since=2021-02-19&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=false)
+
+* **Project management**:
+    * Managed releases `v1.2` and `v1.3` (2 releases) on GitHub
+    * Managed the issue tracker and milestones v1.1 - v1.4 (4 milestones) on GitHub.
+
+* **Enhancements to existing features**:
+    * Reafactored AB3 code to be adapted for Taskify (Pull requests 
+      [\#31](https://github.com/AY2021S2-CS2103T-W14-4/tp/pull/31))
+    * Wrote additional tests for existing features to increase coverage.
+
+* **Documentation**:
+    * User Guide:
+        * Added glossary `delete` and `find`: [\#72]()
```

Comment:
Link to github not included

Reply:
I am still updating haha. Can help merge this version first

Reply:
Alright sure